tests/integration/suite/placement/apilevel/no_max.go

```diff
@@ -20,9 +20,12 @@ import (
 	"testing"
 	"time"
 
+	placementtests "github.com/dapr/dapr/tests/integration/suite/placement/shared"
```

Comment:
Please move to `github.com/dapr/dapr/tests/integration/framework/placement`.

I see a lot of the funcs coming from this package can be members of `Placement` itself.

Reply:
I had the same feeling.